localnet/manage.sh

```diff
@@ -136,6 +136,9 @@ build-localnet() {
     fi
   done
 
+  mkdir -p /tmp/chainflip/
+  touch /tmp/chainflip/debug.log
```

Comment:
We should use the variable we have `$DEBUG_OUTPUT_DESTINATION`

Reply:
updated